SECURITY.md

```diff
@@ -1,15 +1,39 @@
 # Security Policy
 
+## Reporting a Vulnerability
+
+Please report any security issues via email to security@confio.gmbh. 
+
+You will receive a response from us within 48 hours. If the issue is confirmed, we will release a patch as soon as possible depending on complexity but historically within a few days.
```

Comment:
48h is the weekend but I guess somebody is on call

Reply:
good point, I made it 2 working days. Better